src/Riok.Mapperly/Symbols/IMappableMember.cs

```diff
@@ -22,7 +23,11 @@ public interface IMappableMember
 
     bool CanSet { get; }
 
+    bool CanAccessiblySet { get; }
+
     bool IsInitOnly { get; }
 
     bool IsRequired { get; }
+
+    ExpressionSyntax Build(ExpressionSyntax source, bool nullConditional = false);
```

Comment:
IMO `BuildAccess` would be a better name here as this doesn't really build the member itself but only access to it.